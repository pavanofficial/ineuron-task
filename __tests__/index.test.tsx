import { render, screen } from '@testing-library/react'
import "@testing-library/jest-dom";
import Home from '../src/pages/index'

it('renders homepage unchanged', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})

describe("Home", () => {
    it("renders the stock market scan listing", () => {
      render(<Home />);
      expect(screen.getByTestId("cardBox")).toBeInTheDocument();
      expect(screen.getByTestId("listing")).toBeInTheDocument();
    })
})