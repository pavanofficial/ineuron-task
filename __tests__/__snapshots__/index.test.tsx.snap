// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main
    class="flex flex-col justify-center items-center p-6 bg-gray-100 min-h-[100vh]"
  >
    <div
      class="max-w-sm rounded-md overflow-hidden shadow-lg bg-white justify-self-center min-w-[34vw]"
      data-testid="cardBox"
    >
      <ul
        class="divide-y divide-gray-200"
        data-testid="listing"
        role="list"
      />
    </div>
  </main>
</div>
`;
